EKF/estimator_interface.cpp

```diff
@@ -517,30 +545,15 @@ bool EstimatorInterface::initialise_interface(uint64_t timestamp)
 	      _output_buffer.allocate(_imu_buffer_length) &&
 	      _output_vert_buffer.allocate(_imu_buffer_length))) {
 
-		ECL_ERR_TIMESTAMPED("buffer allocation failed!");
+		printBufferAllocationFailed("");
 		unallocate_buffers();
 		return false;
 	}
 
-	_dt_imu_avg = 0.0f;
-
-	_imu_sample_delayed.delta_ang.setZero();
-	_imu_sample_delayed.delta_vel.setZero();
-	_imu_sample_delayed.delta_ang_dt = 0.0f;
-	_imu_sample_delayed.delta_vel_dt = 0.0f;
 	_imu_sample_delayed.time_us = timestamp;
 
-	_initialised = false;
-
-	_time_last_imu = 0;
-	_time_last_gps = 0;
-	_time_last_mag = 0;
-	_time_last_baro = 0;
-	_time_last_range = 0;
-	_time_last_airspeed = 0;
-	_time_last_optflow = 0;
 	_fault_status.value = 0;
-	_time_last_ext_vision = 0;
```

Comment:
Already initialized through initialization brackets {} in the `estimator_interface.hpp`